#
# Submitted-By: Marc Kleine-Budde, 2006-06-01
# Committed-By: Robert Schwebel
#
# Error:
#
# Although the ACE/TAO kit was configured with --disable-gperf, it is
# being built. 
#
# Description:
#
# The Makefile.am in apps includeds the gperf subdirectory
# unconditionally; it should do it according to the result of
# COMPILE_GPERF. This patch fixes the Makefile.am, although it should
# really fix the mwc source this file was generated from. As we don't
# know anything about mwc, submit this to upstream and find somebody who
# knows about this tool.
#
# State:
# 
# unknown, to be submitted
#

diff -urN ACE-5.5.1+TAO-1.5.1-orig/apps/Makefile.am ACE-5.5.1+TAO-1.5.1-ptx/apps/Makefile.am
--- ACE-5.5.1+TAO-1.5.1-orig/apps/Makefile.am	2004-11-12 23:28:16.000000000 +0100
+++ ACE-5.5.1+TAO-1.5.1-ptx/apps/Makefile.am	2006-06-01 17:23:58.000000000 +0200
@@ -8,12 +8,19 @@
 ## MPC Command:
 ## /acebuilds/ACE_wrappers-repository/bin/mwc.pl -include /acebuilds/MPC/config -include /acebuilds/MPC/templates -feature_file /acebuilds/ACE_wrappers-repository/local.features -noreldefs -type automake -exclude build,Kokyu
 
+OPTDIRS =
+
+if COMPILE_GPERF
+OPTDIRS += gperf
+endif
+
+
 SUBDIRS = \
         Gateway \
         JAWS \
         JAWS3 \
         drwho \
-        gperf \
         mkcsregdb \
-        soreduce
+        soreduce \
+	$(OPTDIRS)
 
